src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphCorrect.java

```diff
@@ -9,15 +9,15 @@
 package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocparagraph;
 
 /**
- * Some Javadoc. // ok
```

Comment:
Please remove the trailing whitespace here.

Reply:
@MANISH-K-07 place a code 'request change' instead would more useful in review. The contributor can directly apply the changes then.

Reply:
@abhishekmaity  yes sir it definitely helps lot !!